charts/linkerd2/templates/linkerd-values.yaml

```diff
@@ -0,0 +1,33 @@
+{{ if or (.Values.tracing.enabled) -}}
+{{- $dupValues := .Values -}}
+{{- $dupValues := omit $dupValues "partials" -}}
```

Comment:
Is this really needed? If I omit this line I still get the same result.
Could you also prepend a comment to this section to clarify what's going on?

Reply:
Yep Correct, It was a previous change that came it. Removed now.